.github/workflows/markdown-table-workflow/index.js

```diff
@@ -16,51 +16,94 @@ const verboseRuntimes = {
   swift: "Swift",
 };
 
-const folderDenylist = [".github", ".git"];
-
-const generateUniqueTemplates = (runtimes) => {
-  let templates = [];
-
-  for (const runtime of runtimes) {
-    const folders = fs
-      .readdirSync(path.join(".", `../../../${runtime}`), {
-        withFileTypes: true,
-      })
-      .filter((dirent) => dirent.isDirectory())
-      .map((dirent) => dirent.name);
-
-    templates.push(...folders);
-  }
-
-  return [...new Set(templates)];
+const verboseTemplates = {
+  "Analyze With Perspectiveapi": "Analyze With PerspectiveAPI",
+  "Generate Pdf": "Generate PDF",
+  "Prompt Chatgpt": "Prompt ChatGPT",
+  "Push Notification With Fcm": "Push Notifications With FCM",
+  "Url Shortener": "URL Shortener",
+  "Whatsapp With Vonage": "WhatsApp With Vonage",
```

Comment:
Can we instead have list of words? Like

```
"Pdf": "PDF",
"Chatgpt": "ChatGPT",
"Url: "URL"
...
```

Reply:
Also I think we need same for staying in title-case, Words like `a`, `with`, `to` and others needs to stay lowercased. I think you can copy this list from some title-case library

Reply:
done - updated screenshot also